### DISCUSSION TITLE
Import a file specified by environment variable

### DISCUSSION THREAD

**Discussion by MariuszCwikla** [casey/just#1972] [Q&A]
I can't find example with `import` (or `mod`) + string interpolation.
I would like to do something like this:

```
mylibpath := env('MYLIB_PATH') + "/mylib.just"
import "{{mylibpath}}"
```

**Comment by afeldkamp** [comment]
I wish this were supported on other justfile settings as well.  Here is my specific use case and why this feature would be extremely valuable to me.

I have multiple .env files with vars for different GCP projects, following the naming convention `.<project-id>.env`.
I have a just var that will capture the current project-id that looks like this:
```
current_project := `gcloud config get project`
```
Now, I am trying to do something like:
```
set dotenv-filename := ".{{ current_project }}.env"
```
...hoping I could dynamically load the correct .env file based on the current project, but this unfortunately doesn't work.

**Comment by casey** [comment]
#1849, in-progress, fixes this.
